### PR TITLE
Add Adyen Giving sample - Donation flow

### PR DESCRIPTION
**Pull request by Kwok-he-Chu**

### CODE REVIEW

File: giving-example/Controllers/ApiController.cs [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+            {
+                MerchantAccount = _merchantAccount, // Required.
+                Reference = orderRef.ToString(), // Required.
+                Channel = PaymentRequest.ChannelEnum.Web,
+                Amount = new Amount("EUR", 10000), // Value is 100€ in minor units.
+                
+                // Required for 3DS2 redirect flow.
+                ReturnUrl = $"{_urlService.GetHostUrl()}/api/handleShopperRedirect?orderRef={orderRef}",
+
+                // Used for klarna, klarna is not supported everywhere, hence why we've defaulted to countryCode "NL" as it supports the following payment methods below:
```
Comment (gcatanese) [inline giving-example/Controllers/ApiController.cs:96]: We can remove Klarna as it is not used in this sample

File: giving-example/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <h1>Adyen Giving - Donations demo</h1>
+    <div class="giving-view-container">
+    <p>
+        This is a sample designed to demonstrate the <a href="https://docs.adyen.com/online-payments/donations/web-component">Adyen Giving</a> donations workflow.
+    </p>
+    <p>
+        First make a test payment using one of our <a href="https://docs.adyen.com/development-resources/test-cards/test-card-numbers">test card numbers</a>, so you can see the donation screen appear.
+    </p>
+    <p>
+        To learn more about Adyen's giving feature, check out <a href="https://docs.adyen.com/online-payments/donations/">donations.</a>
```
Comment (gcatanese) [inline giving-example/Views/Home/Index.cshtml:15]: I have removed this in Node. Isn't one link to `https://docs.adyen.com/online-payments/donations` enough?

File: giving-example/Views/Home/Result.cshtml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+@{
+    ViewData["Title"] = @ViewBag.Status;
+}
+
```
Comment (gcatanese) [inline giving-example/Views/Home/Result.cshtml:4]: In Node I have added a `</br>` here so the page title and the success icon don't overlap

**Review by gcatanese — CHANGES_REQUESTED** [review]
Good work, only minor comments. And please wait as we rework Node sample app (store donation token in the backend)

File: giving-example/Views/Home/Index.cshtml (hunk 2 above)
Comment (Kwok-he-Chu) [reply giving-example/Views/Home/Index.cshtml:15]: I've removed this line

**Review by gcatanese — APPROVED** [review]

**Pull request merged**